CHANGELOG.md

```diff
@@ -193,6 +193,9 @@ Changelog
 * RADIUS
   * radius_global (@jonnytpuppet)
   * radius_server (@jonnytpuppet)
+
+* PRIAVATE-VLAN
```

Comment:
Please move this under the `Unreleased` section.

https://github.com/davidefdl/cisco-network-node-utils/blob/feature/private-vlan/CHANGELOG.md#unreleased